### DISCUSSION TITLE
npm ERR! code EINVALIDTAGNAME when installing package

### DISCUSSION THREAD

**Discussion by nik-lampe** [johnlindquist/kit#753] [Errors]
This morning I fired up kit to write a new script, tried to import a npm package and suddenly it did not work any more..

This is the error I'm getting:

```
Installing lodash and continuing...
☠️ ERROR PROMPT SHOULD SHOW ☠️
Error: Command failed with exit code 1: npm i lodash && echo '

cmd+enter to continue script...'
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "&&" of package "&&": Tags may not have any characters that encodeURIComponent encodes.
```

The script is really just installing the package:

```
import "@johnlindquist/kit";

const { _ } = await npm("lodash");
```

Node / npm version is 17.7.1 / 8.5.2 (also happens on 16.4.0 / 8.1.4)

Same happens when I just go `kit install lodash`

Worked fine all the time before and I can't find any issue for this. Maybe it's my machine. Maybe not. 
Any help? ☺️

**Comment by nik-lampe** [comment]
Update:

This only happens when I run the script from the cli.
When I run it via the kit overlay, it installs the package.

**Comment by johnlindquist** [comment]
Ah crap, sorry about that. I'll get that fixed in this weekend's release.

Essentially, the UI uses the built-kit Kit terminal when doing an `npm i` which prints a message "cmd+enter to continue" after it has completed. The CLI is supposed to skip that message if the `process.env.KIT_CONTEXT === "app"`, but apparently I missed something somewhere. I'll dig in and get it sorted. Here's the relevant line if you're curious:

https://github.com/johnlindquist/kit/blob/main/src/cli/install.ts#L20
